releases/release-1.23/README.md

```diff
@@ -14,19 +14,19 @@ description: |
 
 * [This document](https://git.k8s.io/sig-release/releases/release-1.23/README.md)
 * [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.23/release-team.md)
-* [Meeting Minutes](http://bit.ly/k8s122-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
+* [Meeting Minutes](http://bit.ly/k8s123-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
 * [v1.23 Release Calendar][k8s123-calendar]
 * Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
-* [Internal Contact Info][Internal Contact Info] (accessible only to members of [kubernetes-release-team@])
+* [Internal Contact Info][http://bit.ly/k8s123-contacts] (accessible only to members of [kubernetes-release-team@])
```

Comment:
Let's update this in the links section at the bottom of the document instead. (Line 93)

Reply:
also: s/`[kubernetes-release-team@]`/`[release-team@]` here and on line 103